apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -1562,6 +1562,61 @@
                         }
                     }
                 },
+                "currency" : {
+                    "type" : "object",
+                    "additionalProperties" : false,
+                    "properties" : {
+                        "long_name": {
+                            "description": "Long name of currency.",
+                            "type": "string",
+                            "default": "aeternity"
+                        },
+                        "short_name": {
+                            "description": "Short name of currency.",
+                            "type": "string",
+                            "default": "AE"
+                        },
+                        "smallest_denomination" : {
```

Comment:
Alternative names would be `subunit` or `minor_units`

- https://en.wikipedia.org/wiki/Denomination_(currency)
- https://english.stackexchange.com/questions/223121/is-there-a-word-for-the-most-granular-parts-of-currency-eg-cents-pence-etc